docs/pages/admin-guides/access-controls/access-request-plugins/access-request-plugins.mdx

```diff
@@ -33,7 +33,7 @@ The following Access Request plugins are hosted on Teleport Cloud:
 
 ## Self-hosting Access Request plugins
 
-You can host Teleport Access Request plugins yourself. Self-hosted Access
+You can host Teleport Zero Trust Access Request plugins yourself. Self-hosted Access
```

Comment:
My 2c: this reads like a sales brochure and not like natural conversation. Can we just say "access request plugins"?